src/app/app.service.ts

```diff
@@ -30,6 +30,7 @@ export class AppService implements OnModuleInit, OnApplicationBootstrap {
     this.logger.log(`DRY RUN ${this.configService.get('DRY_RUN') ? 'enabled' : 'disabled'}`);
     this.logger.log(`Slot time: ${this.configService.get('CHAIN_SLOT_TIME_SECONDS')} seconds`);
     this.logger.log(`Epoch size: ${this.configService.get('FETCH_INTERVAL_SLOTS')} slots`);
+    this.logger.log(`Effective Balance: ${this.configService.get('EFFECTIVE_BALANCE_INCREMENTS')} ETH`);
```

Comment:
I would say that it is not worth logging. This parameter affects only one particular part of the application while the rest described above are more general